chukwuemekachm/utility_manager_web
# Xrole: implement dashboard navigation

#### What does this PR achieve?
 This PR handles dashboard navigation within the app 

#### Any background context?
- [x] extend icon component to handle click event
- [x] pass history object from dashboard down to navigation component
- [x] create navigation component
- [x] add function to toggle navigation display
- [x] add action to handle page to navigate

#### Manual Testing Steps
- From the dashboard page:
  - click on the hamburger menu
  - select any of `Dashboard`, `Profile`, `Create New Organisation`, `Password` 
  - this should navigate you to their respective pages
  - click on `Logout` 
  - this should log you out from the app

#### Screenshots:

<img width="400" alt="Screenshot 2020-02-15 at 23 52 53" src="https://user-images.githubusercontent.com/32772323/74596350-688d8e00-504e-11ea-9da0-48c586e5af6c.png">

#### Relevant Trello Tickets
- [Finishes[#92]](https://trello.com/c/hYFBcLLg/92-d-2-implement-navigation-for-dashboard)
